### PR TITLE
Fix: Removing type failure and Restructring ambassadors page

### PR DESCRIPTION
**Pull request by DhairyaMajmudar**
**What kind of change does this PR introduce?**
1. Fixed import statements
2. Fixed the location of ambassadors' component files
3. Renamed type definitions and files with more sensible names
4. Improved LinkedIn icon
5. Removed unwanted code and comments 
6. Removed warnings

**Issue Number:**
N/A

### CODE REVIEW

File: components/AmbassadorsCard.tsx [hunk 1]
```diff
@@ -151,9 +157,8 @@ const AmbassadorCard: React.FC<AmbassadorCardProps> = ({ ambassador }) => {
           </p>
         )}
 
-        {/* Social Media Links */}
         <div className='flex justify-center mb-4'>
-          {socialPlatforms.map((platform) => {
+          {SocialIconss.map((platform) => {
```
Comment (DarhkVoyd) [inline components/AmbassadorsCard.tsx:161]: Small typo

**Review by DarhkVoyd — COMMENTED** [review]
Hey, the `SocialIcons` type is redundant, it's should be simply inferred from `SocialIconss` object.

File: pages/ambassadors/index.page.tsx [hunk 2]
```diff
@@ -2,19 +2,19 @@ import React from 'react';
 import { getLayout } from '~/components/SiteLayout';
 import { SectionContext } from '~/context';
 
-import ambassadorsBanner from '../../public/img/community/ambassadors.png';
-import AmbassadorBanner from './components/AmbassadorBanner';
-import ambassadorData from '../../data/ambassadors.json';
-import AmbassadorCard from './components/AmbassadorCard';
-import ambassadorList from '../../data/ambassador_lists.json';
+import ambassadorData from '~/data/ambassadors.json';
```
Comment (DarhkVoyd) [inline pages/ambassadors/index.page.tsx:5]: Importing this way is causing the workflow to fail due to TypeScript compiler type checking, the file should be imported using `fs` and `getStaticProps`.

**Review by benjagm — APPROVED** [review]
Thanks for this!!

**Pull request merged**